sormas-backend/src/main/java/de/symeda/sormas/backend/docgeneration/QuarantineOrderFacadeEjb.java

```diff
@@ -0,0 +1,294 @@
+package de.symeda.sormas.backend.docgeneration;
```

Comment:
I use the exception for mostly validation of user input her. Can you suggest an appropriate exception for this which is already implemented, or suggest a name for one to be implemented?

Reply:
I think it depends on the specific cause for the exception. E.g. in line 199, an IllegalArgumentException would make sense because the fileName does not look like we expect it to look like. If an IOException is thrown, I don't think there's not necessarily a need to encapsulate it in a custom exception. Generally, the methods in QuarantineOrderFacadeEjb do a lot of different things and finding a name for an exception that fits all these things seems pretty hard, but, again, I don't think it's necessary that a class only throws a single exception in all cases.

Reply:
Exception types adapted.